clarify selective sync backfill behaviours (#7418)
segmentio/segment-docs: src/connections/storage/warehouses/faq.md

Selective Sync helps manage the data Segment sends to each warehouse, allowing you to sync different sets of data from the same source to different warehouses.

> When you disable a source, Segment no longer syncs data from that source, and Segment won’t delete any historical data from your data warehouse. When you re-enable a source, Segment will sync all events since the last successful data warehouse sync automatically.

Reviewer: ```suggestion
When you disable a source, Segment no longer syncs data from that source. The historical data from the source remains in your warehouse, even after you disable a source. When you re-enable a source, Segment will automatically sync all events since the last successful data warehouse sync.
```

> When you disable a collection or a property, the automatic backfill behaviour doesn’t apply when that collection or property is re-enabled in the future. You will only find new data generated after re-enabling a collection or property synced into your warehouse. To recover the data, please submit a ticket to backfill the skipped data.

Reviewer: ```suggestion
When you disable and then re-enable a collection or a property, Segment does not automatically backfill the events since the last successful sync. The only data in the first sync following the re-enabling of a collection or property is any data generated after you re-enabled the collection or property. To recover any data generated while a collection or property was disabled, please reach out to [friends@segment.com](mailto:friends@segment.com).
```
